include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -421,12 +460,14 @@ attributes too, for example,
       ArrayRefParameter<"unsigned">:$paddings,
       // Order of logical tensor dimensions; fastest-varying first.
       ArrayRefParameter<"unsigned">:$order,
-      "CTALayoutAttr":$CTALayout
+      "CTALayoutAttr":$CTALayout,
+      DefaultValuedParameter<"std::optional<LinearLayout>", "{}">:$linearComponent
```

Comment:
Should we assert that
1. The linearComponent does not have the `CTALayout` already baked in?
2. The order is trivial if the linearComponent is provided?